wopmars/tests/test_wopmars.py

```diff
@@ -199,6 +199,15 @@ def test_forceall_dryrun(self):
         self.assertEqual(se.exception.code, 0)
 
     def test_dry_run(self):
+        cmd_line = ['python', '-D', self.__db_url, '-w', self.__example_def_file2_only_files, '-v']
```

Comment:
For this PR and all others, can you please check whether you actually need all of these lines?

You can start by deleting everything after Line 207 for this PR or maybe deleting all added lines besides Line 210.